src/pudl/transform/eiaaeo.py

```diff
@@ -514,6 +514,70 @@ def core_eiaaeo__yearly_projected_generation_in_end_use_sectors_by_fuel_type(
     return renamed_for_pudl
 
 
+@asset(io_manager_key="pudl_io_manager")
+def core_eiaaeo__yearly_projected_delivered_energy(
+    raw_eiaaeo__energy_consumption_by_sector_and_source,
+):
+    """Projected delivered energy."""
+    sanitized = filter_enrich_sanitize(
+        raw_df=raw_eiaaeo__energy_consumption_by_sector_and_source,
+        relevant_series_names=(
+            "Energy Use : Commercial : Delivered Energy",
+            "Energy Use : Industrial : Delivered Energy",
+            "Energy Use : Residential : Delivered Energy",
+            "Energy Use : Transportation : Delivered Energy",
```

Comment:
Might be nice to make an issue with the other series in case someone else wants to integrate them.

Reply:
(idk if you already did this, but I went ahead and fleshed out the existing #4228 issue to include the other series'. Let me know if this is conflicting with anything else you've already created! 

Also, that would mean that this PR doesn't actually close that issue, but rather, a small component of that issue.

Reply:
Oh shoot I'd made #4277 for that